src/renderer/VideoButton.tsx

```diff
@@ -393,160 +283,119 @@ export default function VideoButton(props: IProps) {
 
   const getDownloadButton = () => {
     return (
-      <Tooltip title="Download to disk">
-        <IconButton
+      <Tooltip content="Download to disk">
+        <Button
           onMouseDown={stopPropagation}
           onClick={downloadVideo}
-          sx={iconButtonSx}
+          variant="secondary"
+          size="icon"
         >
-          <DownloadIcon sx={{ color: 'white' }} />
-        </IconButton>
+          <CloudDownload />
+        </Button>
       </Tooltip>
     );
   };
 
-  const getShareableLinkSnackBarSuccess = () => {
-    return (
-      <SnackBar
-        message="Link copied, valid for up to 30 days."
-        timeout={2}
-        open={linkSnackBarSuccessOpen}
-        setOpen={setLinkSnackBarSuccessOpen}
-        color="#bb4420"
-      />
-    );
-  };
-
-  const getShareableLinkSnackBarFailed = () => {
-    return (
-      <SnackBar
-        message="Failed to generate link, see logs."
-        timeout={2}
-        open={linkSnackBarFailedOpen}
-        setOpen={setLinkSnackBarFailedOpen}
-        color="#ff0033"
-      />
-    );
-  };
-
   const getShareableLink = async (event: React.MouseEvent<HTMLElement>) => {
     event.stopPropagation();
     event.preventDefault();
 
     try {
       await ipc.invoke('getShareableLink', [videoName]);
-      setLinkSnackBarSuccessOpen(true);
+      toast({
+        title: 'Share link generated',
```

Comment:
Wording not quite right here: "Link generated" / "Shareable link generated" would be fine. Although it's not totally obvious it's been copied to the clipboard so I think we want to get that information in here too.

Reply:
Changed toast title to `Shareable link generated and placed in clipboard`